zipkin-lens/babel.config.js

```diff
@@ -0,0 +1,18 @@
+const presets = [
+  [
+    '@babel/env',
```

Comment:
We can use shorthand name (whole name is `@babel/preset-env`).

Please see:
* https://babeljs.io/docs/en/v7-migration#package-renames